docs/rules/no-unused-expressions.md

```diff
@@ -59,3 +59,10 @@ delete a.b
 ```js
 void a
 ```
+
+
+Additionally in ECMAScript 6, when using a generator, this is also valid:
+
+```js
+yield 0;
```

Comment:
This isn't necessary. Flagging `yield` was a bug, there's no need to call this out in the docs.